### ISSUE TITLE
Sometimes update doesn't get to the firmware update stage

### ISSUE THREAD

**Issue by johnnyman727** [tessel/t2-cli#320]
It freezes on updating the Tessel:
```
➜  ~  t2 update
INFO Looking for your Tessel...
INFO Connected to Tessel-02A3DEABD7E4 over LAN
WARN Could not find firmware version on Tessel-02A3DEABD7E4
WARN Forcefully updating...
INFO Updating Tessel-02A3DEABD7E4 to latest version (0.0.4)...
INFO Beginning update download. This could take a couple minutes..
  downloading [====================] 100% 0.0s remaining
INFO Download complete!
INFO Updating OpenWRT (1/2)
INFO Transferring image of size 19.92 MB
INFO Starting OpenWRT update. Please do not remove power from Tessel.

```

**Comment by johnnyman727** [comment]
The issue is that OpenWRT is inconsistent with what gets printed to `stdout` from the `sysupgrade` command. 

When this issue arises it's because all that gets piped to `stdout` is
```
INFO Starting OpenWRT update. Please do not remove power from Tessel.
Saving config files...
Sending TERM to remaining processes ... logd rpcd netifd odhcpd dbus-daemon mdnsd spid mDNSResponder dnsmasq ntpd ubusd
Sending KILL to remaining processes ...
Switching to ramdisk...
Performing system upgrade...
```

But when it works properly, we get:
```
Saving config files...
Sending TERM to remaining processes ... logd rpcd netifd odhcpd dbus-daemon mdnsd spid mDNSResponder ntpd dnsmasq ubusd
Sending KILL to remaining processes ...
Switching to ramdisk...
Performing system upgrade...
Upgrade completed
Rebooting system...
```

I'm not sure what could cause the inconsistent upgrades. It's possible that the newer OpenWRT versions changed what's printed but I'm not sure yet.

**Comment by johnnyman727** [comment]
This was hopefully fixed in `0.0.18`. Haven't seen it in a long time.
